webpack.prod.config.js

```diff
@@ -13,7 +13,8 @@ var config = {
     publicPath: "/",
     filename: "reusable-components.js",
     library: "ReusableComponents",
-    libraryTarget: "umd"
+    libraryTarget: "umd",
+    globalObject: 'typeof self !== \'undefined\' ? self : this'
```

Comment:
What's this for?

Reply:
That was for trying to get rid of the webpack config errors.  I don't think it does anything; I'll double-check, but it can probably go away.